OpenRA.Game/Map/CellLayer.cs

```diff
@@ -142,11 +111,6 @@ public bool Contains(MPos uv)
 			return bounds.Contains(uv.U, uv.V);
 		}
 
-		public CPos Clamp(CPos uv)
-		{
-			return Clamp(uv.ToMPos(GridType)).ToCPos(GridType);
-		}
-
```

Comment:
Is there a specific reason to remove this? Even if it isn't used right now, it seems a bit odd that every method here has an overload for both `CPos` and `MPos` except this one method.

Reply:
Restored it.